presto-product-tests/bin/product-tests-suite-1.sh

```diff
@@ -2,12 +2,12 @@
 
 set -xeuo pipefail
 
-exit_code=0
+tests_exit_code=0
```

Comment:
maybe `suite_exit_code` would be even better? -- less ambiguous, no chances of accidental reuse